janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/formats/util/GiraphRecordReader.java

```diff
@@ -50,6 +59,12 @@ public GiraphRecordReader(final GiraphInputFormat.RefCountedCloseable<JanusGraph
     @Override
     public void initialize(final InputSplit inputSplit, final TaskAttemptContext taskAttemptContext) throws IOException, InterruptedException {
         reader.initialize(inputSplit, taskAttemptContext);
+
+        final Configuration conf = HadoopCompat.getConfiguration(taskAttemptContext);
```

Comment:
Use taskAttemptContext.getConfiguration(), not the HadoopCompat method which comes from Cassandra package.

Reply:
Thanks for your review! I totally overlooked HBaseSnapshotInputFormatIT.